components/modal/modal.js

```diff
@@ -228,6 +238,13 @@ Modal.propTypes = {
    * Determine whether to close the modal dialog when clicked on overlay.
    */
   closeOnOverlayClick: PropTypes.bool,
+  /**
+   * Data attribute. You can use it to set up any custom data-* attribute.
+   */
+  dataAttrs: PropTypes.oneOfType([
+    PropTypes.bool,
```

Comment:
For me it's not really clear why we support boolean as a proptype for dataAttrs. But it's already implemented this way, probably there is a reason for that. Just curious... 🤔